frontend/src/components/WorkflowSamplesTable/WorkflowSamplesTable.tsx

```diff
@@ -34,9 +34,10 @@ export interface WorkflowSamplesTableProps {
 		selectedSampleIDs: FMSId[],
 		onSelectionChanged: (selectedSamples: SampleAndLibrary[]) => void
 	}
+	loading?: boolean
```

Comment:
Is this optional required ? this appear to not be optional (and an optional "loading" is likely to lead to weird behaviour).